binding.gyp

```diff
@@ -70,10 +83,23 @@
           'outputs': [
             '<@(examples_idl_output_files)',
           ],
-          'action': [
-            '<@(PRODUCT_DIR)/../../bootstrap/command/node',
-            '<@(PRODUCT_DIR)/generator/main.js',
-            '<@(_inputs)'
+          'conditions': [
+            ['OS!="win"',
+            {
+              'action': [
+                '<@(PRODUCT_DIR)/../../bootstrap/command/node',
+                '<@(PRODUCT_DIR)/generator/main.js',
+                '<@(_inputs)'
+              ],
+            }],
+            ['OS=="win"',
+            {
+              'action': [
+                '<@(PRODUCT_DIR)/../../third_party/node/nodevars.bat',
```

Comment:
This looks something wrong. The nodevars.bat will just set environment variables for node.
Why do you call this here? IMHO, we should run the actual node binary instead.

Reply:
i faced error when i used node binary and i changed to use nodevars.bat without see details.
actually when nodeva.bat used instead of node, it was not working.
and i figured out why i faced error first time.

```
CUSTOMBUILD : error : template not found: ../template/example.njk [C:\github\bacardi\build\idl.vcxproj]
      at createTemplate (C:\github\bacardi\node_modules\nunjucks\src\environment.js:207:31)
      at next (C:\github\bacardi\node_modules\nunjucks\src\lib.js:210:13)
      at handle (C:\github\bacardi\node_modules\nunjucks\src\environment.js:253:25)
      at C:\github\bacardi\node_modules\nunjucks\src\environment.js:264:21
      at next (C:\github\bacardi\node_modules\nunjucks\src\lib.js:207:13)
      at Object.exports.asyncIter (C:\github\bacardi\node_modules\nunjucks\src\lib.js:214:5)
      at new_cls.getTemplate (C:\github\bacardi\node_modules\nunjucks\src\environment.js:243:17)
      at new_cls.render (C:\github\bacardi\node_modules\nunjucks\src\environment.js:305:14)
      at Object.module.exports.render (C:\github\bacardi\node_modules\nunjucks\index.js:72:14)
      at C:\github\bacardi\build\Release\generator\main.js:88:46
```
even though i set absolute path, it was also makes same error.
so i leave comment in gyp and i made to not run nunjucks.render.